playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -36,9 +44,21 @@ def oom_killer_enricher(event: NodeEvent):
     oom_kills: List[List] = []
     headers = ["time", "pod", "container", "image"]
     for pod in results.items:
-        oom_statuses = filter(is_oom_status, pod.status.containerStatuses)
-        for status in oom_statuses:
-            dt = parse_kubernetes_datetime_to_ms(status.lastState.terminated.finishedAt)
+        container_id_to_oom_killed_state = {}
+
+        oom_statuses_from_state = filter(is_state_in_oom_status, pod.status.containerStatuses)
+        for status in oom_statuses_from_state:
+            container_id_to_oom_killed_state[status.state.terminated.containerID] = (status, status.state)
+
+        oom_statuses_from_last_state = filter(is_last_state_in_oom_status, pod.status.containerStatuses)
+        for status in oom_statuses_from_last_state:
+            container_id_to_oom_killed_state[status.lastState.terminated.containerID] = (status, status.lastState)
+
+        for v in container_id_to_oom_killed_state.values():
+            status = v[0]
```

Comment:
should be `containerID` not `status` right?

can you just make this `for (containerID, state) in container_id_to_oom_killed_state.values()`?